ingestion/functions/parsing/brazil_espirito_santo/espirito_santo.py

```diff
@@ -0,0 +1,326 @@
+                    "end": convert_date(date_death)
+                },
+                "value": "Death",
+            }
+        )
+    if "Cura" in outcome:
+        events.append(
+            {
+                "name": "outcome",
+                "value": "Recovered",
+            }
+        )
+    return events
+
+
+def convert_test(pcr: str, serological_method1: str, serological_method2: str, serological_method3: str, test_class: str):
+    items = (serological_method1, serological_method2, serological_method3)
+    if pcr:
+        return "PCR test"
+    elif any(item not in _NONE_TYPES for item in items):
+        return "Serological test"
+    elif "Clinico" in test_class:
+        return "Clinical diagnosis"
+
+
+def convert_preexisting_conditions(lung: str, kidney: str, diabetes: str,
+                                   cardiovascular: str, obesity: str):
+    preexistingConditions = {}
+    items = (lung, kidney, diabetes, cardiovascular, obesity)
+    if all(item in _NONE_TYPES for item in items):
+        return None
+
+    preexistingConditions["hasPreexistingConditions"] = True
+    comorbidities = []
+
+    if lung == "Sim":
+        comorbidities.append(_COMORBIDITIES_MAP["ComorbidadePulmao"])
+    if kidney == "Sim":
+        comorbidities.append(_COMORBIDITIES_MAP["ComorbidadeRenal"])
+    if diabetes == "Sim":
+        comorbidities.append(_COMORBIDITIES_MAP["ComorbidadeDiabetes"])
+    if cardiovascular == "Sim":
+        comorbidities.append(_COMORBIDITIES_MAP["ComorbidadeCardio"])
+    if obesity == "Sim":
+        comorbidities.append(_COMORBIDITIES_MAP["ComorbidadeObesidade"])
+
+    if comorbidities:
+        preexistingConditions["values"] = comorbidities
+
+    return preexistingConditions
+
+
+def convert_symptoms(headache: str, throat: str, fever: str, cough: str, diarrhoea: str, dyspnea: str):
+    symptoms = {}
+    items = (headache, throat, fever, cough, diarrhoea, dyspnea)
+    if all(item in _NONE_TYPES for item in items):
+        return None
+
+    symptoms["status"] = "Symptomatic"
+    values = []
+
+    if headache == "Sim":
+        values.append(_SYMPTOMS_MAP["Cefaleia"])
+    if throat == "Sim":
+        values.append(_SYMPTOMS_MAP["DorGarganta"])
+    if fever == "Sim":
+        values.append(_SYMPTOMS_MAP["Febre"])
+    if cough == "Sim":
+        values.append(_SYMPTOMS_MAP["Tosse"])
+    if diarrhoea == "Sim":
+        values.append(_SYMPTOMS_MAP["Diarreia"])
+    if dyspnea == "Sim":
+        values.append(_SYMPTOMS_MAP["DificuldadeRespiratoria"])
+
+    if values:
+        symptoms["values"] = values
+
+    return symptoms
+
+
+def convert_ethnicity(ethnicity: str):
+    if ethnicity == "Preta":
+        return "Black"
+    elif ethnicity == "Parda":
+        return "Mixed"
+    elif ethnicity == "Amarela":
+        return "Asian"
+    elif ethnicity == "Branca":
+        return "White"
+    elif ethnicity == "Indigena":
+        return "Indigenous"
```

Comment:
@Mougk is "indigenous" expected here? I would have thought that we'd need more context (the indigenous Brazilians being different people from the indigenous Ainu in Japan, for example).